From ce13bc2b50c0b5687f81d622f5897fd072d7faf5 Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Tue, 7 Jun 2016 15:25:22 +0200
Subject: [PATCH] 2

---
 init.rc | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/init.rc b/init.rc
index cc7001e..73ff068 100755
--- a/init.rc
+++ b/init.rc
@@ -775,3 +775,10 @@ service logcatd /system/bin/logcat -b all -v threadtime -v usec -v printable -D
     user logd
     group log
     writepid /dev/cpuset/system-background/tasks
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class main
+    user root
+    group root
+    oneshot    
-- 
1.9.1

